src/importexport/midi/internal/midiimport/importmidi_simplify.cpp

```diff
@@ -62,6 +62,8 @@ bool areDurationsEqual(
         sum += ReducedFraction(d.second.fraction()) / d.first;
     }
 
+    // Note: This looks really weird, but fixing it seems to break several
```

Comment:
It seems that this function is used only for debugging. I believe we should just remove it completely because it does more harm than good

Reply:
Yes, it is used for debugging only, And there causes issues when fixed to do a usefull comparison.